release_notes/ocp_3_5_release_notes.adoc

```diff
@@ -0,0 +1,1683 @@
+the `service.alpha.openshift.io/serving-cert-secret-name` API and regenerate them.
+
+Set the `service.alpha.openshift.io/serving-cert-secret-name` to the name you
+want to use for your secret. Then, your `PodSpec` can mount that secret. When it
+is available, your pod will run. The certificate will be good for the internal
+service DNS name, `<service.name>.<service.namespace>.svc`. The certificate and
+key are in PEM format, stored in *_tls.crt_* and *_tls.key_*, respectively.
+
+----
+$ oc get secret ssl-key -o yaml
+
+kind: Secret
+metadata:
+  annotations:
+    service.alpha.openshift.io/expiry: 2017-03-19T08:07:07Z
+----
+
+When the regenerator finds a certificate that does not have the expiry
+annotation, it will regenerate as well. However, the existing secret is not
+invalidated. Therefore, no manual intervention is required to get the
+regeneration behavior.
+
+See xref:../dev_guide/secrets.adoc#service-serving-certificate-secrets[Service Serving Certificate Secrets] for more information.
+
+[[ocp-35-configurable-expiry-range-for-framework-certs]]
+==== Configurable Expiry Range for Framework Certificates
+
+By default, the certificates used to govern the etcd, master, and kubelet expire
+after two to five years. There is now an `oc` command to change this expiry to
+be end-user configurable. This has not been implemented in Ansible yet.
+
+Use the `oadm ca` command, specifying a validity period greater than two years:
+
+----
+# oadm ca create-master-certs --hostnames=example.org --signer-expire-days=$[365*2+1]`
+----
+
+See
+xref:../install_config/master_node_configuration.adoc#creating-new-configuration-files[Creating
+New Configuration Files] for more information.
+
+[[ocp-35-can-i-command-and-scc-review-command]]
+==== can-i Command and scc-review Command Options
+
+The `can-i` and `scc-review` command options allow users to better understand
+their permissions and
+xref:../architecture/additional_concepts/authorization.adoc#security-context-constraints[security
+context constraints (SCC)] setting in their projects. Users see a list of the
+commands they are allowed to execute.
+
+The `can-i` command  option tests scopes in terms of the user and role. The
+`scc-review` command option checks which `ServiceAccount` can create a pod.
+
+`scc-subject-review` can check whether a user or a `ServiceAccount` can create a
+pod.
+
+List which permissions a particular user or group has in the project by project
+administrator:
+
+----
+$ oc policy can-i --list --user=**
+$ oc policy can-i --list --groups=**
+----
+
+List which permissions a particular user or group has in the project by system
+administrator role:
+
+----
+$ oc policy can-i --list --user=** -n project
+$ oc policy can-i --list --groups=**  -n project
+----
+
+Determine if users can have all the combination of verbs and resources from `oc
+policy can-i --list --user/group`
+
+----
+$ oc policy can-i <verb> <resource> --user/group
```

Comment:
I think the "group" is actually "--groups" (plural). From oc 3.5:

http://pastebin.test.redhat.com/473925

How bout for all of the "--user/group", instead have:

```
[--user|--groups]
```
Showing that it's optional, and that you have a choice between the two?